### ISSUE TITLE
Upgrade 2.0.0. to 2.0.1 deletes files in pub/media/import

### ISSUE THREAD

**Issue by robfico** [magento/magento2#3097]
Installed tar.gz for 2.0.0 with sample data with web root of /pub. I put images in pub/media/import.

Ran following commands to upgrade to 2.0.1:

```
vi composer.json (change 2.0.0 to 2.0.1 for "magento/product-community-edition")

touch vendor/magento/magento2-base/dev/tests/js/JsTestDriver/.gitignore
touch vendor/magento/magento2-base/dev/tests/api-functional/.gitignore
touch vendor/magento/magento2-base/dev/tests/unit/.gitignore
touch vendor/magento/magento2-base/dev/tests/static/.gitignore
touch vendor/magento/magento2-base/dev/tests/integration/.gitignore
touch vendor/magento/magento2-base/dev/tests/functional/.gitignore

composer update
```

After upgrade, all image files in pub/media/import were gone. Not sure if this should happen or not, but I would assume it should not.

**Comment by robfico** [comment]
On upgrading from 2.0.1 to 2.0.2 via "composer update" the upgrade process does delete the pub/media/import folder and then re-create it empty. Whatever is in pub/media/import does get deleted.

**Comment by mbahar** [comment]
I cloned the repo to install 2.0.0 , and after a 'composer update' all pub/media deleted.

**Comment by PieterCappelle** [comment]
Had the same problem today. Doing composer update removed the pub/media/import folder.

**Comment by KrystynaKabannyk** [comment]
Hello @robfico, this issue has been fixed in the 2.1.0 Release, that's why I'm closing it. If you any questions or additional information regarding the issue feel free to reopen it or create a new one.
